### PR TITLE
hide proposal system temporarily

### PR DESCRIPTION
**Pull request by SivanYeh**
Hide proposal system temporarily before bugs fixed.
Add parameter that makes hiding proposal system easier.
More specifically, hide or remove links for items with red frame as following:
image
image

### CODE REVIEW

File: components/core/header/nav-bar/NavBarHamburger.vue [hunk 1]
```diff
@@ -82,6 +82,7 @@
                 @click.native="toggleAccordion('venue')"
             ></nav-bar-item-accordion>
             <ext-link
+                v-if="showProposalSystem"
```
Comment (baby230211) [inline components/core/header/nav-bar/NavBarHamburger.vue:85]: Is this working???
due to you're using showProposalSystem flag,
we should add this property in computed.
something like this
```js
showProposalSystem() {
            return this.$store.state.configs.showProposalSystem
        },
```

Comment (SivanYeh) [reply components/core/header/nav-bar/NavBarHamburger.vue:85]: > Is this working??? due to you're using showProposalSystem flag, we should add this property in computed. something like this
> 
> ```js
> showProposalSystem() {
>             return this.$store.state.configs.showProposalSystem
>         },
> ```

Thanks for reviewing! I'm uploading it on staging and checking(my local deploy is not working currently

Comment (SivanYeh) [reply components/core/header/nav-bar/NavBarHamburger.vue:85]: @baby230211 It does work. 
PyCon Taiwan
I learnt it from switches in store/index.js. Dug in a bit, it is started from a helpful commit from Josix: https://github.com/pycontw/pycontw-frontend/commit/3c2d0429c4f99e15cb7a56b30e77da91eadfc48f

**Review by mattwang44 — APPROVED** [review]
LGTM

**Review by baby230211 — APPROVED** [review]
LGTM

**Pull request merged**